### ISSUE TITLE
channeled abilities getting interrupted randomly, hellfire and r...

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#4160]
> channeled abilities getting interrupted randomly, hellfire and rain of fire when in combat inside dungeon

.go xyz -260.205000 -187.845000 -51.874800 732 2.733090

**Comment by Shadownflame** [comment]
https://github.com/Project-Epoch/bugtracker/issues/3351

**Comment by Bennylavaa** [comment]
had this happen a few times during the beta it is one of the bigger issues ive run into seeing it seems to affect all channeled stuff.

**Comment by Shadownflame** [comment]
Same, however I had trouble duplicating it on internal. I know Eredun fixed Arcane missiles now, I wonder if other channels also have an innate property to miss on cast?

**Comment by ghost** [comment]
I don't believe it's related to the Arcane Missiles issue, if it was the same issue you'd be able to see a message in the combat log about why it failed, as these two spells don't have the flag that prevents that feedback like Arcane Missiles did. I am also not having any luck replicating this issue
